aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -252,6 +253,15 @@ module aptos_framework::object {
         create_object_internal(owner_address, unique_address, false)
     }
 
+    /// Create an object at a specific address. Used by framework.
+    public(friend) fun create_object_at_address(
+        owner_address: address,
+        object_address: address,
+        can_delete: bool
```

Comment:
this would not be deletable under current rules, because it contains a static guid

Reply:
That's why I called with `false` and it is the only callsite? do you think I should change the signature of this function? I don't quite mind actually.

Reply:
well if we have no need for anything but false, we should keep the functions signature clean... after all it is malleable after this since it is friend

Reply:
sgtm